pkg/api/universal/lock.go

```diff
@@ -126,27 +130,28 @@ type tryLockUnlockRequest interface {
 // Internal method that checks if the request is for a lock store component.
 func (a *Universal) lockValidateRequest(req tryLockUnlockRequest) (lock.Store, error) {
 	var err error
+	lockType := string(contribMetadata.LockStoreType)
 
 	if a.compStore.LocksLen() == 0 {
-		err = messages.ErrLockStoresNotConfigured
+		err = apiErrors.NotConfigured(req.GetStoreName(), lockType+" store", nil, codes.FailedPrecondition, http.StatusInternalServerError, "ERR_LOCK_STORE_NOT_CONFIGURED", kitErrors.CodePrefixLock+kitErrors.CodeNotConfigured)
```

Comment:
The component type should be "lock," but in this context, "lock store" is more appropriate. so, I included the word "store."